### PR TITLE
Added add fraction test with denominator zero

### PR DESCRIPTION
**Pull request by sumitshyamsukha**

### CODE REVIEW

File: DumbMathTest.java [hunk 1]
```diff
@@ -61,4 +61,13 @@ public void testMultiplyTwoNegativeNumbers() {
 		assertEquals(expected, actual, 0.00000000001);
 	}
 
+	@Test(expected = IllegalArgumentException.class)
+	public void testAddFractionDenominatorZero() {
+		int n1 = 1;
+		int d1 = 0; 
+		int n2 = 1;
```
Comment (nfallen) [inline DumbMathTest.java]: can you change these names to something more descriptive?

Comment (sumitshyamsukha) [inline DumbMathTest.java]: Added commit with descriptive variable names.

**Comment by ChrisMurphyOnline** [comment]
Nova, you can merge this when you're okay with it

**Comment by nfallen** [comment]
LGTM
